### PR TITLE
Adds a printer definition for the FLSUN QQ

### PR DESCRIPTION
**Pull request by radfahrer**
This is my first attempt to add a new printer to Cura. I think I've followed all the instructions, but Cura tells me that the profile is corrupt. Perhaps someone could help me.

### CODE REVIEW

**Comment by fieldOfView** [comment]
In both files, `version ` has to be `2`.

**Comment by radfahrer** [comment]
Why 2?

**Comment by fieldOfView** [comment]
It refers to the version of the format, not the version of the data.

File: resources/definitions/flsun_qq.def.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		},
+		"machine_width": {
+			"default_value": 240
+		},
+		"machine_height": {
+			"default_value": 285
+		},
+		"machine_depth": {
+			"default_value": 240
+		},
```
Comment (Ghostkeeper) [inline resources/definitions/flsun_qq.def.json]: [This page](https://all3dp.com/1/flsun-qq-3d-printer-review/) claims that the printer has a printing area of 260x260x370mm. You might want to review this. Or did you experimentally find that the usable print area is smaller?

File: resources/definitions/flsun_qq.def.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	"version": 2,
+	"name": "FLSUN QQ",
+	"inherits": "fdmprinter",
+	"metadata": {
+		"visible": true,
+		"manufacturer": "FLSUN",
+		"author": "Daniel Green",
+		"file_formats": "text/x-gcode",
+		"icon": "icon_ultimaker2",
+		"platform_offset": [0, 0, 0],
```
Comment (Ghostkeeper) [inline resources/definitions/flsun_qq.def.json]: This offset is useless if there is no platform mesh set. It's also set to the default value. It doesn't hurt to have it in here, but you might as well remove it.

**Comment by Ghostkeeper** [comment]
Developers, see issue CURA-6250.

File: resources/definitions/flsun_qq.def.json (hunk 2 above)
Comment (fieldOfView) [reply resources/definitions/flsun_qq.def.json]: icon is useless as well; it does not exist and it is not used

File: resources/definitions/flsun_qq.def.json (hunk 1 above)
Comment (radfahrer) [reply resources/definitions/flsun_qq.def.json]: These are the settings recommended by the owners manual.

**Comment by radfahrer** [comment]
I've removed the two fields that were pointed out as not necessary.

**Comment by Ghostkeeper** [comment]
Thanks. Seems to work fine, then.

**Pull request merged**